src/qt/rpcconsole.cpp

```diff
@@ -45,6 +45,7 @@
 #include <QTime>
 #include <QTimer>
 
+#include <string>
```

Comment:
My bad, I meant [`#include <util/string.h>`](https://github.com/bitcoin/bitcoin/blob/master/src/util/string.h) for the `Join` template function.

Reply:
Oops, fixed